### PR TITLE
Update debug-services doc

### PR DESCRIPTION
**Pull request by thockin**
I have been meaning to update this for a long time.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6243)

### CODE REVIEW

File: docs/tasks/debug-application-cluster/debug-service.md [hunk 1]
```diff
@@ -263,8 +265,9 @@ debugging your own `Service`, debug DNS.
 
 ## Does the Service work by IP?
 
-The next thing to test is whether your `Service` works at all.  From a
-`Node` in your cluster, access the `Service`'s IP (from `kubectl get` above).
+Assuming we can confirm that DNS works, the next thing to test is whether your
+`Service` works at all.  From a `Node` in your cluster, access the `Service`'s
```
Comment (tengqm) [inline docs/tasks/debug-application-cluster/debug-service.md]: change `Node` to *Node* ?

File: docs/tasks/debug-application-cluster/debug-service.md [hunk 2]
```diff
@@ -388,6 +394,20 @@ this is not what happens (or whatever the correct behavior is for your own
 `kubectl logs` to be useful or `kubectl exec` directly to your `Pods` and check
 service from there.
 
+Another thing to check is that your `Pods` are not crashing or being restarted.
```
Comment (tengqm) [inline docs/tasks/debug-application-cluster/debug-service.md]: `Pods` -> `Pod`s

**Comment by thockin** [comment]
Updated, all comments addressed

**Pull request merged**